ui/src/app/shared/components/application-selector/application-selector.tsx

```diff
@@ -18,7 +18,7 @@ export const ApplicationSelector = ({apps, formApi}: {apps: models.Application[]
                     <label key={app.metadata.name} style={{marginTop: '0.5em', cursor: 'pointer'}}>
                         <CheckboxField field={`app/${i}`} />
                         &nbsp;
-                        {app.metadata.name}
+                        {app.isAppOfAppsPattern ? `(App of Apps)${app.metadata.name}` : app.metadata.name}
```

Comment:
Add a space after the right bracket ')'

Reply:
As discussed, we could probably redesign this so that the app of apps are listed out separately under a different list or category, so it's even more obvious.  This can be done later in a separate PR.

Reply:
Thanks for the review. Added space after the right bracket in this PR.